recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "cuda-nvcc-split" %}
-{% set version = "12.1.105" %}
-{% set cuda_version = "12.1" %}
+{% set version = "12.2.140" %}
+{% set cuda_version = "12.2" %}
```

Comment:
Could we parse this from `version`?

Reply:
https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/8#discussion_r1433272238